AirportTicketBookingSystem/AirportTicketBookingSystem/Utilities/ToSeatClass.cs

```diff
@@ -0,0 +1,18 @@
+﻿using AirportTicketBookingSystem.Models;
+
+namespace AirportTicketBookingSystem.Utilities
+{
+    public static class SeatClassExtensions
```

Comment:
the file name should be `SeatClassExtensions` not `ToSeatClass`. Should match the class name.